test/run

```diff
@@ -25,9 +25,15 @@ case $TEST_SCENARIO in
         test/common/run-tests --jobs ${TEST_JOBS:-1} --test-dir test/verify --track-naughties
         ;;
     pybridge)
-        test/image-prepare --verbose $TEST_OS
-        make -C tmp/build-dist/ cockpit-bridge.pyz
-        bots/image-customize -v --upload tmp/build-dist/cockpit-bridge.pyz:/usr/bin/cockpit-bridge $TEST_OS
+        rm -rf tmp/wheel
+        tools/systemd_ctypes
+        python3 -m build --no-isolation --wheel --outdir tmp/wheel
+        bots/image-customize --verbose \
+            --fresh \
+            --install python3-pip \
+            --upload tmp/wheel:/var/tmp \
+            --run-command 'pip install --prefix=/usr /var/tmp/wheel/*' \
```

Comment:
Why do we have to call pip install inside the image? Isn't it enough to just copy it to some place in `$PYTHONPATH`? Also, this needs some extra commands to ensure that this does not access the internet. In fact `image-customize` option` --no-network` is missing in a lot of places here in test/run (we added it to e.g. podman and machines, but apparently forgot it in cockpit?)

Reply:
See discussion in https://github.com/cockpit-project/bots/pull/4068 . I'm happy enough with `--no-network` after landing that. Until then, good enough.